### PR TITLE
2018-detroit-program

### PR DESCRIPTION
**Pull request by archieco**

### CODE REVIEW

File: content/events/2018-detroit/program.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
++++
+Title = "Program"
+Type = "program"
+Description = "Program for devopsdays detroit 2018"
++++
+
+<div class = "row">
+  <div class = "col">
+    <hr />
+    If Open Space is new to you, you may be interested in <a href="/pages/open-space-format">more details about Open Space</a>.
```
Comment (InAnimaTe) [inline content/events/2018-detroit/program.md]: `If Open Spaces are new to you`

File: content/events/2018-detroit/speakers/armon-dadgar.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
++++
+Title = "Armon Dadgar"
+Twitter = "armon"
+image = "armon-dadgar.png"
```
Comment (InAnimaTe) [inline content/events/2018-detroit/speakers/armon-dadgar.md:4]: so this actually works? seems you're providing a full path on the talk page; do we really have to specify images twice? :(

File: content/events/2018-detroit/program/tom-kowalski.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
++++
+Talk_date = ""
+Talk_start_time = ""
+Talk_end_time = ""
+Title = "Devops In a Serverless World"
+Type = "talk"
```
Comment (InAnimaTe) [inline content/events/2018-detroit/program/tom-kowalski.md:6]: Is there any differentiator b/w presentation and ignite?

File: content/events/2018-detroit/speakers.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
++++
+Title = "Speakers"
+Type = "speakers"
+Description = "Speakers for devopsdays detroit 2018"
```
Comment (InAnimaTe) [inline content/events/2018-detroit/speakers.md]: Depending how this is displayed, may want to ensure proper capitalization

File: data/events/2018-detroit.yml [hunk 5]
```diff
@@ -107,3 +107,194 @@ sponsor_levels:
...
+    type: custom
+    date: 2018-10-03
+    start_time: "08:30"
+    end_time: "09:30"
+  - title: "Opening Welcome"
+    type: custom
+    date: 2018-10-03
+    start_time: "09:30"
+    end_time: "09:45"
+  - title: "armon-dadgar"
```
Comment (InAnimaTe) [inline data/events/2018-detroit.yml:122]: So the Title here isn't the title for the talk but references the individual file with the title above (I'm guessing)?

File: data/events/2018-detroit.yml [hunk 6]
```diff
@@ -33,8 +33,8 @@ nav_elements: # List of pages you want to show up in the navigation of your page
   - name: propose
```
Comment (smithtrevor) [inline data/events/2018-detroit.yml]: can comment this out as proposals aren't being accepted anymore.  Would also need to remove from the welcome page.

File: content/events/2018-detroit/program.md (hunk 1 above)
Comment (archieco) [reply content/events/2018-detroit/program.md]: right... this was part of the generated text from the scripts in `utilities`...  maybe we can fix ours and suggest another change separately.

File: content/events/2018-detroit/speakers/armon-dadgar.md (hunk 2 above)
Comment (archieco) [reply content/events/2018-detroit/speakers/armon-dadgar.md:4]: correct

File: content/events/2018-detroit/program/tom-kowalski.md (hunk 3 above)
Comment (archieco) [reply content/events/2018-detroit/program/tom-kowalski.md:6]: yep, not sure if you mean in general or this file specifically. But, ignites are rendered differently in the output.

File: data/events/2018-detroit.yml (hunk 5 above)
Comment (archieco) [reply data/events/2018-detroit.yml:122]: Yes, this is how it is wired together with devopsdays' hugo customizations.

File: content/events/2018-detroit/program.md (hunk 1 above)
Comment (InAnimaTe) [reply content/events/2018-detroit/program.md]: Sounds good.

**Review by InAnimaTe — APPROVED** [review]

File: data/events/2018-detroit.yml (hunk 6 above)
Comment (archieco) [reply data/events/2018-detroit.yml]: done

**Pull request merged**